Automattic/pocket-casts-android
# Add content description for "in up next"

## Description
Adds a contentDescription to the up next icon in the podcast episode list. This is needed so screen readers will read if the episode is queued for up next. I made some changes to https://github.com/Automattic/pocket-casts-android/pull/754

Fixes https://github.com/Automattic/pocket-casts-android/issues/750

## Testing Instructions

- Enable Talkback in android accessibility settings
- Go to the episode list
- Ensure the content description contains information for: In up first, downloading, downloading, and in progress

## Screenshots or Screencast 
[Screenshot](https://user-images.githubusercontent.com/6628497/232632546-4e4c1aec-7aa1-41d3-9508-c9b389c72c76.png)

## Checklist
I don't think it's needed?
 
#### I have tested any UI changes...
